tests/integration/slashing.go

```diff
@@ -658,6 +682,7 @@ func (suite *CCVTestSuite) TestQueueAndSendSlashPacket() {
 	}
 
 	// send all pending packets - only slash packets should be queued in this test
+	// TODO: the following call can be removed
```

Comment:
If we can remove, then why don't we do it here? Is it for reasons of clarity we want to keep it?

It might make sense to create issues for all TODOs we introduce in code and then in the TODO comment we can have something like:
```
// TODO (issue XYZ): Remove the following call
```
This way we can prevent cases were we forget to go back and fix the TODOs. (cc Not sure what's your take on that @mpoke)

Reply:
In this case, I think this was just a reminder for future shawn to remove the call, that I missed :) see https://github.com/cosmos/interchain-security/commit/569599a425d8fd9bdf8c5a1dd2fca313be8ce247

In general I agree that TODO comments are most useful when there's an issue attached to it